org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/resources/bpmn4tosca/bppmn4tosca.json

```diff
@@ -1,116 +1,193 @@
 [
-  {
```

Comment:
ok, I will keep the original test data unchanged and add a new test case for exclusive gateway.

By the way, I found the test data file name is "bppmn4tosca.json". there are two 'p' in the name. Is this a spell mistake or you name it on purpose?  If this is  a spell mistake, shall I change the file name to 'bpmn4tosca.json'.